R/utils.R

```diff
@@ -14,14 +14,6 @@ check_2d_matrix <- function(x) {
   if (anyNA(x)) {
     cli::cli_abort("Input must be a matrix with no missing values.")
   }
-
-  if (any(x[, 1L] >= x[, 2L])) {
-    cli::cli_abort("Birth values must be stricly less than death values.")
-  }
```

Comment:
I would leave this test.

Reply:
OK, i was thinking of the #4 thread, and i think we're on the same page. The check should stay, but it should warn rather than abort, with a tag/flag affixed to the result.

Reply:
Yes sorry I was not clear enough. I am going to implement this.